extra/innochecksum.cc

```diff
@@ -1263,6 +1282,12 @@ innochecksum_get_one_option(
 	const char		*IF_DBUG(argument,),
 	const char *)
 {
+	/* Tablespace flags doesn't have short option */
+	if (strcmp(opt->name, "tablespace-flags") == 0) {
+		use_tablespace_flags = true;
+		return false;
+	}
```

Comment:
Can we please validate the flags here? And use a special (invalid) value of `tablespace_flags`, say, `~0`, to indicate that the parameter has not been specified by the user.